custom_student_verification/signals.py

```diff
@@ -52,3 +58,15 @@ def reject_user(user, reason):
     ManualVerification.objects.filter(user=user, status='approved').delete()
     # send an email to the user about verification rejection.
     send_verification_rejected_email(user, reason)
+
+
+@receiver(post_save, sender=get_course_enrollment_model())
+def auto_verify_paid_students(sender, instance, created, **kwargs):  # pylint: disable=unused-argument
+    """
+    Signal helps to auto-verify student who are paying for the course.
+    This signal will only take effect when the switch is enabled.
+    """
+    course_mode = get_course_mode_model()
+    if settings.FEATURES.get('ENABLE_PAID_COURSE_AUTO_VERIFY'):
```

Comment:
@farhaanbukhsh, should we only do it when the `created` flag is `True`?

Reply:
yes, we should only do this when the created flag is true. Else the user should have the normal flow.

Reply:
@farhaanbukhsh, so could you add `created` to the if condition? 
```suggestion
    if created and settings.FEATURES.get('ENABLE_PAID_COURSE_AUTO_VERIFY'):
```

Reply:
@shimulch  there is a bit of issue with created, what if learner is enrolled in the `audit` mode and just the enrollment mode is changed to `verified` so that time the model is not `created` rather updated. Hence I feel the code as is make sense.

What do you say?

Reply:
@farhaanbukhsh, yeah, that makes sense. My concern was about multiple verification emails being sent. But it turns out it already got [covered](https://github.com/open-craft/custom-student-verification/blob/61117d2abb585f9a7897ff115c14806ac2b1a71c/custom_student_verification/signals.py#L32).